esignet-service/.env.example

```
@@ -33,3 +33,20 @@ REDIS_READ_TIMEOUT=3s
+CLIENT_ADDITIONAL_CONFIG_SCHEMA_URL=
+
+# ── Scope-based JWT authorization ────────────────────────────────────────────
+# Empty AUTH_JWKS_URI disables auth (startup warning logged). Set both for
+# production. Tuning knobs (refresh, timeout, clock skew, audience) are
+# documented in README.
+#
+# Typical shape for a Keycloak-backed deployment:
+#   AUTH_JWKS_URI=https://<keycloak-host>/auth/realms/<realm>/protocol/openid-connect/certs
+#   AUTH_ISSUER=https://<keycloak-host>/auth/realms/<realm>
+AUTH_JWKS_URI=
+AUTH_ISSUER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_ISSUER key should go before the AUTH_JWKS_URI key

(UnorderedKey)